### PR TITLE
Score Rendering improvements

### PR DESCRIPTION
**Pull request by storrisi**
This is related to #652 

This is how the tooltip looks like after the suggested changes:
Schermata 2019-05-14 alle 18 36 54

This implementation still miss one of the suggested changes, about link the titles to a document, that is not yet be written.

### CODE REVIEW

**Comment by storrisi** [comment]
I've also added a link to the documentation which completes and fixes #652

File: src/components/Navigation/Ui/ScoreRenderer.js [hunk 1]
```diff
@@ -71,28 +72,56 @@ class ScoreRenderer extends Component {
...
+          </h2>
           {this.renderScore(tools)}
         </div>
       </div>
     )
   }
 
+  renderUnit = () => (
+    <span>
+      / 100 <span className="unit">points</span>
```
Comment (geneh) [inline src/components/Navigation/Ui/ScoreRenderer.js]: Let's drop "points" everywhere and make sure the spacing around the slash is even.

File: src/components/Navigation/Ui/ScoreRenderer.js [hunk 2]
```diff
@@ -71,28 +72,56 @@ class ScoreRenderer extends Component {
...
+    </span>
+  )
+
+  renderScoreInfo = () => (
+    <div className="ScoreInfo">
+      <a
+        href="https://github.com/clearlydefined/license-score/blob/master/ClearlyLicensedMetrics.md#clearlylicensed-scoring-formula"
+        target="_blank"
+      >
+        What does these scores mean?
```
Comment (geneh) [inline src/components/Navigation/Ui/ScoreRenderer.js]: Let's change it to "Scoring formula"

**Comment by geneh** [comment]
👍

**Pull request merged**